tests/test_ops/test_roi_align.py

```diff
@@ -40,6 +43,8 @@
 def _test_roialign_gradcheck(device, dtype):
     if not torch.cuda.is_available() and device == 'cuda':
         pytest.skip('test requires GPU')
+    if is_mlu() and device == 'mlu':
+        pytest.skip('test requires MLU')
```

Comment:
```suggestion
```

Reply:
We have used the following lines to check whether cuda is available()
```python
pytest.param(
        'cuda',
        marks=pytest.mark.skipif(
            not is_cuda(), reason='requires CUDA support')),
```

Reply:
It was removed.